spark/dl/src/main/scala/com/intel/analytics/bigdl/models/autoencoder/Autoencoder.scala

```diff
@@ -35,12 +35,12 @@ object Autoencoder {
     model
   }
 
-  def graph(classNum: Int): Module[Float] = {
+  def graph(classNum: Int, isDynamic: Boolean = false): Module[Float] = {
     val input = Reshape(Array(featureSize)).inputs()
     val linear1 = Linear(featureSize, classNum).inputs(input)
     val relu = ReLU().inputs(linear1)
     val linear2 = Linear(classNum, featureSize).inputs(relu)
     val output = Sigmoid().inputs(linear2)
-    Graph(input, output)
+    if (isDynamic) Graph.dynamic(input, output) else Graph(input, output)
```

Comment:
Why do we need isDynamic for the models/examples? It should always be static graph, no?

Reply:
This is for test... We have a set of test for these graph models, so I want to test both dynamic and static